src/index.html

```diff
@@ -1,13 +1,426 @@
+      <div class="container">
+        <div class="menu__content">
+          <div class="menu__top top-actions">
+            <a href="#" class="icon icon--cross"></a>
+
+            <a href="/" class="logo">
+              <img
+                class="logo__image"
+                src="images/logo.png"
+                alt="Miami Condo Kings"
+              >
+            </a>
+          </div>
+
+          <ul class="menu__list">
+            <li class="menu__item">
+              <a href="#" class="menu__link">Home</a>
+            </li>
+            <li class="menu__item">
+              <a href="#about-us" class="menu__link">About-us</a>
+            </li>
+            <li class="menu__item">
+              <a href="#gallery" class="menu__link">Gallery</a>
+            </li>
+            <li class="menu__item">
+              <a href="#testimonials" class="menu__link">Testimonials</a>
+            </li>
+            <li class="menu__item">
+              <a href="#contact-us" class="menu__link">Contact us</a>
+            </li>
+          </ul>
+
+          <a
+            href="tel:+1 234 555-55-55"
+            class="menu__call menu__call--phone"
+          >
+            +1 234 555-55-55
+          </a>
+          <a
+            href="tel:+1 234 555-55-55"
+            class="menu__call menu__call--action"
+          >
+            call to order
+          </a>
+        </div>
+      </div>
+    </nav>
+
+    <main>
+      <section
+        class="page__section about"
+        id= "about-us"
+      >
+        <div class="container">
+          <div class="grid grid--desktop">
+            <div class="grid__item grid__item--1-4">
+              <h2 class="about__title">
+                Miami best
+                <br>
+                Real estate
+
+                <a href="#" class="about__title-link">here</a>
+              </h2>
+            </div>
+
+            <div class="grid__item grid__item--7-12">
+              <a href="#" class="about__subtitle">
+                Over 50 years experience in love
+              </a>
+
+              <p class="about__paragraph">
+                Mazel and VA Tsukkerman in his "Analysis of musical works." Pointillism, which originated in the music of the early
+                twentieth century, microforms, found a distant historical.
+              </p>
+
+              <p class="about__paragraph">
+                The market leader in real estate since 2003. Awards Real Estate Company, 2010, 2011, 2013. More than 200 satisfied
+                customers premium segment. We focus only on new buildings and high-level doskanalno know this area better than other
+                specialists.Working directly with developers - so have more useful information about the profitable deals.
+              </p>
+
+              <p class="about__paragraph--important">
+                We will never be late!
+                <br>
+                Guaranteed secure transaction - 100%
+              </p>
```

Comment:
This text should be `bold`
![image](https://user-images.githubusercontent.com/37161315/170516196-a59d1b44-197f-413e-b0f2-2e3e6ec5cb1d.png)

Reply:
done